docker-compose.yml

```diff
@@ -1,10 +1,10 @@
 version: '2.1'
 services:
   pg:
-    image: postgres:9.3
+    image: postgres:9.4
 
   conjur_5:
-    image: cyberark/conjur
+    image: registry2.itci.conjur.net/cyberark/conjur
```

Comment:
We'll need to get the supporting version of `cyberark/conjur` published before this PR can go in.  We can't reference internal versions of an image like this in an OS library.

Reply:
Should we also remove it from conjur_4? @jvanderhoof

Reply:
Looks like this still needs to be removed. You can leave v4 alone since it's unrelated to this PR.